modules/dnn/test/test_darknet_importer.cpp

```diff
@@ -218,6 +229,27 @@ TEST_P(Test_Darknet_nets, YOLOv3)
                      classIds, confidences, boxes, scoreDiff, iouDiff);
 }
 
+TEST_P(Test_Darknet_nets, YOLOv3_batch)
```

Comment:
@kopytjuk, I'd like to reuse existing tests. Could you please add a second invocation of `testDarknetModel` with `batch size != 1` to `YoloVoc`, `TinyYoloVoc` and `YOLOv3` tests.
To know which batch size value is failed, please modify
```cpp
        normAssertDetections(refClassIds, refConfidences, refBoxes, classIds,
                             confidences, boxes, "", confThreshold, scoreDiff, iouDiff);
```
to something like
```cpp
        normAssertDetections(refClassIds, refConfidences, refBoxes, classIds,
                             confidences, boxes, format("batch size %d", batch_size).c_str(), confThreshold, scoreDiff, iouDiff);
```